packages/lib/src/document.ts

```diff
@@ -65,19 +67,19 @@ const formatKeyAndContext = (key: string): KeyAndContext => {
   } catch (err) {
     throw new UnsupportedKeyError(key, err)
   }
-  const { jwtAlg, keyBytes } = keyInfo
+  const { jwtAlg } = keyInfo
 
   if (jwtAlg === crypto.P256_JWT_ALG) {
     return {
       context: 'https://w3id.org/security/suites/ecdsa-2019/v1',
```

Comment:
I think we can remove these contexts now as well

Reply:
That is a confusing point to me: they seem like they wouldn't be necessary for JSON-LD validation, in that the key used is sort of hidden in the multibase/multicodec string. But i'm not 100% positive they can be removed entirely. Also, it seems like to be belt-and-suspenders it would be good to have an actual W3C spec that specifies the use of K-256 keys, which doesn't exist yet.

It is pretty easy and harmless to include these if not needed, I think, but also happy to pull them out.

A related open question is whether we need to be using JSON-LD and this `@context` at all. DID Documents are flexible and can use just `application/json` as well (apparently), with possibly some subtle impact on use of fragment identifiers, etc.

We *can* stick with JSON-LD, and it a smaller change to just not touch that, which is why I haven't proposed it, but could be A Thing to revisit at some point.